doc/nrf/glossary.rst

```diff
@@ -558,6 +558,10 @@ Glossary
    Request to Send (RTS)
       In :term:`flow control <Hardware Flow Control (HWFC)>`, the transmitting end is ready and requesting the far end for permission to transfer data.
 
+   Revision
+      A tracked state in time of a Git repository. It can always be
+      represented by a Git SHA, but can also be a Git tag. More info in :ref:`intro_vers_revs` and in `Git revisions`_.
```

Comment:
```suggestion
      A tracked state in time of a Git repository.
      It can always be represented by a Git SHA, but can also be a Git tag.
      For more information, see :ref:`intro_vers_revs` and `Git revisions`_.
```